SBR.md

```diff
@@ -460,7 +461,15 @@ The Random Value SHALL be unique in each email. The Random Value SHALL remain va
 
 The Random Value SHALL be reset upon each instance of the email sent by the CA and, if intended for additional use as an authentication factor, upon first use.  
 
-#### 3.2.2.3 CAA Records
+#### 3.2.2.3 Validating applicant as operator of associated mail server(s)
+
+Confirming the Applicant's control over the rfc822Name email address by confirming control of the SMTP FQDN to which a message delivered to the email address should be directed. The SMTP FQDN MUST be identified using the address resolution algorithm defined in RFC 5321 Section 5.1 which determines which SMTP FQDNs are authoritative for a given email address. If more than one SMTP FQDNs have been discovered, the CA MUST verify control of an SMTP FQDN following the selection process at RFC 5321 Section 5.1.
```

Comment:
RFC 5321, section 5.1 says:
> When a domain name associated with an MX RR is looked up and the
   associated data field obtained, the data field of that response MUST
   contain a domain name.  That domain name, when queried, MUST return
   at least one address record (e.g., A or AAAA RR) that gives the IP
   address of the SMTP server to which the message should be directed.
   Any other response, specifically including a value that will return a
   CNAME record when queried, lies outside the scope of this Standard.
   The prohibition on labels in the data that resolve to CNAMEs is
   discussed in more detail in RFC 2181, Section 10.3 [38].

I think it would be good to clarify here that aliases in MX record RDATA cannot be used for this validation method.

Reply:
May be as simple as adding "Aliases in MX record RDATA MUST NOT be used for this validation method."

Reply:
Thanks, clarified it. I believe we can resolve this.